src/components/install-scratch/install-scratch.jsx

```diff
@@ -13,6 +13,14 @@ const Step = require('../../components/steps/step.jsx');
 
 require('./install-scratch.scss');
 
+const downloadUrls = {
+    mac: 'https://downloads.scratch.mit.edu/desktop/Scratch%20Desktop-3.6.0.dmg',
+    win: 'https://downloads.scratch.mit.edu/desktop/Scratch%20Desktop%20Setup%203.9.0.exe',
+    googlePlayStore: 'https://play.google.com/store/apps/details?id=org.scratch',
+    microsoftStore: 'https://www.microsoft.com/store/apps/9pfgj25jl6x3?cid=storebadge&ocid=badge',
+    macAppStore: 'https://apps.apple.com/us/app/scratch-desktop/id1446785996?mt=12'
+};
+
```

Comment:
👍 on moving into a const like this. In the future this could be a file we load externally with the data and we wouldn't need to deploy www to change the version of the downloads.

Reply:
Yep, exactly! :)